sys/include/unaligned.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_unaligned unaligned access methods
```

Comment:
Since this shows up in top-level doc, I prefer

```suggestion
 * @defgroup    sys_unaligned unaligned memory access methods
```

Reply:
fixed